### PR TITLE
Feature/mtu rssi

### PR DESCRIPTION
**Pull request by pawelByszewski**

### CODE REVIEW

File: android/src/main/java/com/polidea/flutter_ble_lib/delegate/MtuDelegate.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                throw new IllegalArgumentException(methodCall.method + " cannot be handled by this delegate");
+        }
+    }
+
+    private void requestMtu(String deviceIdentifier, @NonNull int mtu, String transactionId, @NonNull final MethodChannel.Result result) {
+        Log.d(TAG, "Request MTU " + mtu);
+        bleAdapter.requestMTUForDevice(deviceIdentifier, mtu, transactionId, new OnSuccessCallback<Device>() {
+            @Override
+            public void onSuccess(Device device) {
+                result.success(device.getRssi());
```
Comment (mikolak) [inline android/src/main/java/com/polidea/flutter_ble_lib/delegate/MtuDelegate.java]: `getRssi`?

File: android/src/main/java/com/polidea/flutter_ble_lib/delegate/RssiDelegate.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        }
+    }
+
+    private void rssi(@NonNull final String deviceIdentifier, String transactionId, @NonNull final MethodChannel.Result result) {
+        Log.d(TAG, "Read rssi for device " + deviceIdentifier + " transactionId: " + transactionId);
+
+        final SafeMainThreadResolver resolver = new SafeMainThreadResolver<>(
+                new OnSuccessCallback<Integer>() {
+                    @Override
+                    public void onSuccess(Integer data) {
```
Comment (mikolak) [inline android/src/main/java/com/polidea/flutter_ble_lib/delegate/RssiDelegate.java]: Rename data, please

File: example/lib/device_details/device_details_bloc.dart [hunk 3]
```diff
@@ -112,6 +113,14 @@ class DeviceDetailsBloc {
         })
         .then((characteristics) => characteristics.forEach((characteristic) =>
             log("Found characteristic \n ${characteristic.uuid}")))
+        .then((_) async {
+            int rssi = await peripheral.rssi();
+            log("rssi $rssi");
+        })
+        .then((_) async {
+          await peripheral.requestMtu(74);
```
Comment (mikolak) [inline example/lib/device_details/device_details_bloc.dart]: We should log negotiated MTU

File: lib/internal/internal_ble_manager.dart [hunk 4]
```diff
@@ -5,6 +5,9 @@ import 'package:flutter_ble_lib/internal/bridge/internal_bridge_lib.dart';
 
 class InternalBleManager
     implements BleManager, ManagerForPeripheral, ManagerForService {
+
+
```
Comment (mikolak) [inline lib/internal/internal_ble_manager.dart]: That's a lot of space here, partner!

File: lib/internal/internal_ble_manager.dart [hunk 5]
```diff
@@ -98,4 +101,20 @@ class InternalBleManager
   @override
   Future<List<Characteristic>> characteristicsForService(Service service) =>
       _bleLib.characteristicsForService(service);
+
+  @override
+  Future<int> rssi(
+      Peripheral peripheral,
+      String transactionId,
+      ) {
+    print("call channel read RSSI");
```
Comment (mikolak) [inline lib/internal/internal_ble_manager.dart]: Should this print stay here?

File: example/lib/device_details/device_details_bloc.dart (hunk 3 above)
Comment (pawelByszewski) [reply example/lib/device_details/device_details_bloc.dart]: will do in another PR

Comment (mikolak) [reply example/lib/device_details/device_details_bloc.dart]: Then create an issue for this on the board

**Pull request merged**